saleor/order/migrations/0175_order_voucher_code.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 3.2.21 on 2023-10-04 10:26
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("discount", "0058_voucher_single_use"),
+        ("order", "0174_order_idx_order_created_at"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="order",
+            name="voucher_code",
+            field=models.ForeignKey(
```

Comment:
You need to create index for this foreign key directly to use concurrently option

Reply:
done